lib/index.js

```diff
@@ -0,0 +1,74 @@
+
+const fs = require('fs');
+const shortid = require('shortid');
+
+class Store {
+  constructor(rootDir) {
+    this.rootDir = rootDir;
+  }
+
+  create(obj, callback) {
+    const _id = shortid.generate();
+    const objWithId = { ...obj, _id };
+    const objWithIdStr = JSON.stringify(objWithId);
+    fs.writeFile(`${this.rootDir}/${_id}`, objWithIdStr, err => {
```

Comment:
```suggestion
    fs.writeFile(`${this.rootDir}/${_id}.json`, objWithIdStr, err => {
```

You need to be saving these objects as json, which they're currently not. To see how your project is working, you can run only your create test (do this by adding ```.only``` after the ```it``` and look in the test data folder.